.github/workflows/verify-node.yml

```diff
@@ -15,6 +15,7 @@ jobs:
           - '12'
```

Comment:
Let's drop running node 12 tests (it goes out of support after april) to avoid making the CI slower